File: Praktikum Modul 4/SOAL-3/BAHASA-C/PRAK403-2410817110014-RIZKI_ADHITIYA_MAULANA.c
#include <stdio.h>

int main ()
{
    int angka_pertama, angka_kedua;

    scanf("%d %d", &angka_pertama, &angka_kedua);

    for (int i = angka_pertama, j = angka_kedua; i <= angka_kedua && j >= angka_pertama; i++, j--)
    {
        printf("%d %d", i, j);
        if (i < angka_kedua||j > angka_pertama)
        {
            printf(" - ");
        }
    }

    for (int i = angka_pertama, j = angka_kedua; i >= angka_kedua && j <= angka_pertama; i--, j++)
    {
        printf("%d %d", i, j);
        if (i > angka_kedua||j < angka_pertama)
        {
            printf(" - ");
        }
    }
    return 0;
}

File: Praktikum Modul 4/SOAL-3/BAHASA-PY/PRAK403-2410817110014-RIZKI_ADHITIYA_MAULANA.py
angka_pertama, angka_kedua = map(int,input("").split())

i = angka_pertama
j = angka_kedua

if i < j:
    while i <= angka_kedua and j >= angka_pertama:
        print(f"{i} {j}", end=" ")

        if i < angka_kedua or j > angka_pertama:
            print("-", end=" ")
        i +=1
        j -=1
    
elif i > j:
    while i >= angka_kedua and j <= angka_pertama:
        print(f"{i} {j}", end=" ")

        if i > angka_kedua or j < angka_pertama:
            print("-", end=" ")
        i -=1
        j +=1